add hyperdrive bindings to the `getPlatformProxy` supported bindings list (#16706)
cloudflare/cloudflare-docs: src/content/docs/workers/wrangler/api.mdx

* [D1 database bindings](/d1/build-with-d1/d1-client-api/)

* [Hyperdrive bindings](/hyperdrive)

  :::note[Hyperdrive values are simple passthrough ones]

>   Values provided by hyperdrive bindings such as `connectionString` and `host` don't have a valid meaning outside of a workerd process,

Reviewer: ```suggestion
  Values provided by hyperdrive bindings such as `connectionString` and `host` do not have a valid meaning outside of a `workerd` process.
```

>   so hyperdrive proxies, instead of returning such values which would be unusable from within node.js, return value correspondent to the

Reviewer: ```suggestion
  This means that Hyperdrive proxies return passthrough values, or value correspondent to the database connection provided by the user.  Otherwise, it would values which would be unusable from within node.js.
```

>   database connection provided by the user (making hyperdrive proxied values simple passthrough).

Reviewer: ```suggestion
```
